docs/.vuepress/config.js

```diff
@@ -70,10 +70,11 @@ module.exports = {
             title: 'Install',
             path: '/install/',
             children: [
-              'install/ipfs-desktop',
-              'install/command-line',
-              'install/server-infrastructure',
-              'install/ipfs-companion'
+              ['install/ipfs-desktop', 'Desktop application'],
+              ['install/command-line', 'Command-line interface'],
+              ['install/ipfs-updater', 'Updater'],
+              ['install/server-infrastructure', 'Server infrastructure'],
+              ['install/ipfs-companion', 'Browser add-on']
```

Comment:
@jessicaschilling , requesting your review specifically for these lines (although feel free to take a look at the rest of the PR if you fancy). As [mentioned in my comment](https://github.com/ipfs/ipfs-docs/pull/538#issuecomment-724323615), this PR changes the item titles within the sidebar, under the **Install** section. Are these changes OK from your side of things?

Reply:
Thanks for checking in 😊 A few thoughts ...

- Is there a major reason not to say "IPFS Desktop" and "IPFS Companion"? They are official product names, so it's good practice to refer to them by official name whenever possible (otherwise that's like saying "carbonated corn syrup beverage" instead of "Coke"). If you're really concerned about people not understanding what Companion is, `IPFS Companion add-on` disambiguates a bit and still fits in the nav.
- I'd actually move the line for Companion underneath the line for Desktop - since the two are closely coupled functionally, and are GUI-driven tools (so keeping them together keeps that menu as being in a sort of increasing level of technical complexity).
- Maybe `IPFS Updater` instead of just "Updater"? Not feeling super strongly either way, though.

Reply:
> Is there a major reason not to say "IPFS Desktop" and "IPFS Companion

Yeah, my issues are around the names of things. _IPFS Desktop_ is not a _desktop_, it's an _application_ that runs in a desktop environment. Calling the it an _application_ tells the reader, _hey, this is an IPFS application that you have to install_. I'd much prefer to give the project an actual name, like _Commander_ or something, but that's an issue for the `ipfs-shipyard/ipfs-desktop` repo.

The same with IPFS Companion. That name doesn't initially tell readers _what_ the tool does, although as you say it might help to have it listed directly underneath _Desktop application_.

Lastly, things look a bit odd visually when everything has the _IPFS_ prefix:

![image](https://user-images.githubusercontent.com/9611008/98608073-75c36a80-22b8-11eb-8359-09f28c001efa.png)

It feels like this should turn into a larger conversation about the names of things. Should I take these commits out and create a new issue so we can have a more involved discussion?

Reply:
@johnnymatthews At present, IPFS Desktop and IPFS Companion **are** the proper names of the tools (hence their capitalization). They are "actual names", and have been in use as such for some time.

Feel free to open issues in their respective repos to discuss renaming the tools themselves, but we really should be referring to things that have proper names by those proper names.

As "IPFS Update" isn't actually a proper name - just the name of the repo - feel free to call that just "updater" or similar, if the visual repetition bugs you.

(cc @lidel for awareness)

Reply:
Cool beans, I'll leave _Updater_ as it is then, and I'll raise issues in the IPFS Desktop and IPFS Companion repositories. Glad I checked!

Reply:
Thank you!